sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## 12.0.0b1 (XX-XX-XX)
+- Initial Release. Please see the README for information on the new design.
+- Support for ChangeFeedClient: get change feed events by page, get all change feed events, get events in a time range
+
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob-changefeed/README.md)
```

Comment:
do we want to redirect to the documentation instead of read me here?

Reply:
good catch!

Reply:
I will address this in the changelog pr